### PR TITLE
Add Indonesian translation

### PR DESCRIPTION
**Pull request by ArsyiAziz**

### CODE REVIEW

File: phonenumber_field/locale/id/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (francoisfreitag) [inline phonenumber_field/locale/id/LC_MESSAGES/django.po:1]: Can remove this comment.

File: phonenumber_field/locale/id/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <arsyiaziz@gmail.com>, 2021.
+#
+#, fuzzy
```
Comment (francoisfreitag) [inline phonenumber_field/locale/id/LC_MESSAGES/django.po:6]: Can remove this fuzzy comment, the entry is fine.

File: phonenumber_field/locale/id/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-02-05 21:42+0700\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
```
Comment (francoisfreitag) [inline phonenumber_field/locale/id/LC_MESSAGES/django.po:12]: Can use the same date as the POT-Creation-Date.

File: phonenumber_field/locale/id/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+"Last-Translator: Arsyi Syarief Aziz <arsyiaziz@gmail.com>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
```
Comment (francoisfreitag) [inline phonenumber_field/locale/id/LC_MESSAGES/django.po:14]:
```suggestion
"Language-Team: Indonesian <LL@li.org>\n"
```

File: phonenumber_field/locale/id/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,44 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <arsyiaziz@gmail.com>, 2021.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
```
Comment (francoisfreitag) [inline phonenumber_field/locale/id/LC_MESSAGES/django.po:9]: Other locales set this to an empty string.
```suggestion
"Project-Id-Version: \n"
```

File: phonenumber_field/locale/id/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+"POT-Creation-Date: 2021-02-05 21:42+0700\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Arsyi Syarief Aziz <arsyiaziz@gmail.com>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+#: phonenumber_field/formfields.py:27
```
Comment (francoisfreitag) [inline phonenumber_field/locale/id/LC_MESSAGES/django.po:20]: Please add an empty line after the headers.
```suggestion

#: phonenumber_field/formfields.py:27
```

**Review by francoisfreitag — COMMENTED** [review]
Other than that, looks fine to me. I totally trust your Indonesian :indonesia:.

File: phonenumber_field/locale/id/LC_MESSAGES/django.po (hunk 6 above)
Comment (ArsyiAziz) [reply phonenumber_field/locale/id/LC_MESSAGES/django.po:19]: Thanks for the corrections. I've revised the format of the file.

**Comment by francoisfreitag** [comment]
Looking good, thanks! Could you squash both commits?

**Comment by ArsyiAziz** [comment]
Sure

**Comment by ArsyiAziz** [comment]
Sorry, have i done it right? This is my first time squashing commits

**Comment by francoisfreitag** [comment]
There should be only one commit at the end of the squashing process.

I usually use the git interactive rebase feature https://docs.github.com/en/github/using-git/about-git-rebase.

Here are the commands I would type:
```console
$ git rebase -i main
# Opens the rebase menu in a text editor, you should see your commits.
# `pick` the first commit, `squash` all following by changing the `pick` command at the beginning of the line to `squash`.
# Save the file and exit the editor.
```

If there are conflicts, solve them, e.g. with `$ git mergetool`.
Once you are satisfied with the result, regenerated the mo file with `$ django-admin compilemessages --locale id`, add the latest compiled messages (e.g. `$ git add -u`) and finish the rebase process with `$ git rebase --continue`.

Here’s the output of `$ git log -p` after I’m done squashing:
<details>
<summary>git log -p</summary>
<pre>
commit 8207f9984e8dfe422c7bcbc93bfcd1766a41117d
Author: ArsyiAziz <arsyiaziz@gmail.com>
Date:   Fri Feb 5 22:15:26 2021 +0700

    Add Indonesian translation

diff --git a/phonenumber_field/locale/id/LC_MESSAGES/django.mo b/phonenumber_field/locale/id/LC_MESSAGES/django.mo
new file mode 100644
index 0000000..43c2396
Binary files /dev/null and b/phonenumber_field/locale/id/LC_MESSAGES/django.mo differ
diff --git a/phonenumber_field/locale/id/LC_MESSAGES/django.po b/phonenumber_field/locale/id/LC_MESSAGES/django.po
new file mode 100644
index 0000000..9b314bc
--- /dev/null
+++ b/phonenumber_field/locale/id/LC_MESSAGES/django.po
@@ -0,0 +1,43 @@
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <arsyiaziz@gmail.com>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-02-05 21:42+0700\n"
+"PO-Revision-Date: 2021-02-05 21:42+0700\n"
+"Last-Translator: Arsyi Syarief Aziz <arsyiaziz@gmail.com>\n"
+"Language-Team: Indonesian <LL@li.org>\n"
+"Language: id\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+
+#: phonenumber_field/formfields.py:27
+#, python-brace-format
+msgid ""
+"Enter a valid phone number (e.g. {example_number}) or a number with an "
+"international call prefix."
+msgstr "Masukkan nomor telepon yang benar (misalkan {example_number}) "
+"atau nomor yang dilengkapi dengan prefiks kode panggilan internasional."
+
+#. Translators: {example_number} is an international phone number.
+#: phonenumber_field/formfields.py:33
+#, python-brace-format
+msgid "Enter a valid phone number (e.g. {example_number})."
+msgstr "Masukkan nomor telepon yang benar (misalkan {example_number})."
+
+#: phonenumber_field/modelfields.py:53
+msgid "Phone number"
+msgstr "Nomor telepon"
+
+#: phonenumber_field/serializerfields.py:9
+msgid "Enter a valid phone number."
+msgstr "Masukkan nomor telepon yang benar."
+
+#: phonenumber_field/validators.py:11
+msgid "The phone number entered is not valid."
+msgstr "Nomor telepon yang dimasukkan tidak benar."

commit 687f0093651e5cd45b5e3c73242a2689588f52fe
Author: François Freitag <mail@franek.fr>
Date:   Sun Jan 17 10:55:25 2021 +0100

    Use default Python version to finish coveralls build

# … 
</pre>
</details>

**Comment by ArsyiAziz** [comment]
Sorry I forced-pushed, I had to remove the the last commit  because it seemed to mess up with the squash
```
Merge: 74ec0e8 dad13d9
Author: ArsyiAziz <arsyiaziz@gmail.com>
Date:   Sun Feb 7 17:17:43 2021 +0700

    Merge branch 'main' of https://github.com/ArsyiAziz/django-phonenumber-field into main
```

**Review by francoisfreitag — APPROVED** [review]
It looks exactly as it should, thanks!
You have to force-push after rewriting the history with git interactive rebase, no worries.

**Pull request merged**